src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt

```diff
@@ -350,33 +350,6 @@ internal class ProtectLevelCalculatorTest {
       validate()
     }
 
-    @Test
```

Comment:
this happens in the client now, by the time we get here the most recent is already the first one